PackageInfo.g

```diff
@@ -111,7 +109,19 @@ ArchiveURL      := Concatenation( ~.SourceRepository.URL,
 Keywords := ["Forms", "Sesquilinear", "Quadratic"],
 
 CommunicatedBy := "Leonard Soicher (London)",
-AcceptDate := "03/2009"
+AcceptDate := "03/2009",
+
+
+AutoDoc := rec(
+    TitlePage := rec(
+        Copyright := Concatenation(
+            "&copyright; 2018 by the authors<P/>\n\n",
```

Comment:
By the way: I copied this straight from the existing manual page. However, I actually think it should be updated:
```suggestion
            "&copyright; 2016-2021 by the authors<P/>\n\n",
```

Reply:
... and by now 2022...